examples/convert_legacy_llama.py

```diff
@@ -806,7 +770,7 @@ class OutputFile:
     def __init__(self, fname_out: Path, endianess:gguf.GGUFEndian = gguf.GGUFEndian.LITTLE):
         self.gguf = gguf.GGUFWriter(fname_out, gguf.MODEL_ARCH_NAMES[ARCH], endianess=endianess)
 
-    def add_meta_model(self, params: Params, metadata: Metadata | None) -> None:
+    def add_meta_model(self, params: Params, metadata: gguf.Metadata | None) -> None:
```

Comment:
Would it be possible to have a method that adds metadata in `utility.py` and share it between `convert_legacy_llama`/`convert_hf_to_gguf`? It'd make sense to have it in one place since any changes should in theory apply to both.